### PR TITLE
Remove old patient search flow

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good. Left few comments, PTAL

File: app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt [hunk 1]
```diff
@@ -214,13 +212,9 @@ class PatientsTabScreen : BaseScreen<
       .map { SimpleVideoClicked }
 
   override fun openPatientSearchScreen(additionalIdentifier: Identifier?) {
```
[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:214]: Yeah, it's doesn't matter TBH. It's still a patient search, right? Ideally

[the remark this answers is not in the document]
Comment (Honey14) [reply app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:214]: Yes it is patient search but when we see open `PatientSearchScreen`, it looks like we're opening patient search screen which does not reflect what we're doing here.

**Review by Honey14 — APPROVED** [review]

File: app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:214]: Eh, we don't have 2 searches anymore. So I think it should be fine 😂

[the remark this answers is not in the document]
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:214]: But if you're concerned with it, I can change it in a different PR.

[the remark this answers is not in the document]
Comment (Honey14) [reply app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:214]: No it's okay. I figured only the `screen` part as a suffix made me concerned. `openPatientSearch` seems fine. This makes sense as well.

**Pull request merged**